app/script/assets/AssetRemoteData.js

```diff
@@ -127,7 +127,7 @@ z.assets.AssetRemoteData = class AssetRemoteData {
           ? z.assets.AssetCrypto.decryptAesAsset(buffer, this.otrKey.buffer, this.sha256.buffer)
           : buffer;
       })
-      .then(plaintext => new Blob([new Uint8Array(plaintext)], {mime_type: type}))
+      .then(plaintext => new Blob([new Uint8Array(plaintext)], {type}))
```

Comment:
This one was wrong (see https://developer.mozilla.org/en-US/docs/Web/API/Blob/Blob)